### ISSUE TITLE
=app-emulation/docker-1.6.0_rc1: could not determine kind of name for C.BTRFS_BUILD_VERSION

### ISSUE THREAD

**Issue by alunduil** [tianon/docker-overlay#38]
```
---> Making bundle: dynbinary (in bundles/1.6.0-rc1/dynbinary)
Created binary: /var/tmp/paludis/app-emulation-docker-1.6.0_rc1/work/docker-1.6.0-rc1/bundles/1.6.0-rc1/dynbinary/dockerinit-1.6.0-rc1
# github.com/docker/docker/daemon/graphdriver/btrfs
could not determine kind of name for C.BTRFS_BUILD_VERSION

!!! ERROR in app-emulation/docker-1.6.0_rc1::docker:
!!! In src_compile at line 4702
!!! dynbinary failed

!!! Call stack:
!!!    * src_compile (/var/tmp/paludis/app-emulation-docker-1.6.0_rc1/temp/loadsaveenv:4702)
!!!    * ebuild_f_compile (/usr/libexec/paludis/2/src_compile.bash:56)
!!!    * ebuild_main (/usr/libexec/paludis/ebuild.bash:668)
!!!    * main (/usr/libexec/paludis/ebuild.bash:691)

diefunc: making ebuild PID 21829 exit with error
die trap: exiting with error.
```

Should be enough information but let me know if you need btrfs versions or anything.

**Comment by tianon** [comment]
Ack, I was hoping we could avoid this one for at least a little while.  A "fix" for this was filed upstream in Docker in https://github.com/docker/docker/pull/11417, but was also discussed in btrfs-progs upstream in https://bugzilla.redhat.com/show_bug.cgi?id=1202373 (and a patch was applied that will add this constant back in the next released version).

On top of that, @vbatts (:heart:) is working on a patch to add this information back conditionally with a nice `#ifdef`. :+1:

So, TL;DR, rc2 should include a fix for this that lets Docker compile properly again and hopefully sometime soon after we'll have an even better fix that gives us this pretty information back too.

**Comment by tianon** [comment]
Care to test rc2 now that it's up, @alunduil? :+1:

**Comment by alunduil** [comment]
Builds fine now.  Will let you know if there are any other issues.

**Comment by tianon** [comment]
:+1: :heart:
